controller/request.go

```diff
@@ -128,6 +138,10 @@ func contains(l []string, e string) bool {
 }
 
 func (c controller) ExtendWithRandomSliceIDs(req Request) (Request, error) {
+	if !req.IsSlicable() {
```

Comment:
this feels ugly, and I fear we might need to put this all over the places. Lets make it working first, than fix the data model, shall we?